test/features/cloudnative/network_zones/network_zones.go

```diff
@@ -53,13 +52,17 @@ func Feature(t *testing.T) features.Feature {
 	builder.WithLabel("name", "cloudnative-network-zone")
 	secretConfig := tenant.GetSingleTenantSecret(t)
 
+	networkZone := getNetworkZoneName()
```

Comment:
Okay we need a third opinion here cc @0sewa0 ?

Reply:
but we are talking implementation details here 😄 

if you give me a deterministic Network zone name, based on some factors, which can be easily reproduced, then you wouldn't need to name it `Random`, because its not that.
- Example: For a given dynakube, your function would return a networkzone name, lets say based on the name of the dynakube and what mode it is in. then `getNetworkZoneName(dk)` is totally fine.

but, when you functions use the `rand` package, then it is kinda important for the user to know, that it will be random.
+1: We are not talking about an external API used as a library somewhere else, where it might be actually irrelevant, but an internal function, used for e2e tests, I wanna know what exactly the e2e tests are doing

Reply:
☝️